client/src/subapps_key_concept_panels/subappps_key_concept_panels.js

```diff
@@ -0,0 +1,26 @@
+import { declare_panel } from "src/panels/panel_declarations/common_panel_utils";
+import {
+  curried_render,
+  common_panel_config,
+} from "src/panels/panel_declarations/misc/key_concept_panels/key_concept_panels";
+
+import { primaryColor } from "src/core/color_defs";
+
+export const declare_estimates_comparison_key_concepts_panel = () =>
+  declare_panel({
```

Comment:
Hmm this is interesting take, I was thinking more like separate 
```
<SomeThingsToKeepInMind>
  <KeyConceptList>
...
```
from `curried_render`, and define it as its own component. Then use that component inside for example `curried_render` and other places (like estimates explorer). I think defining panels for these is a bit overkill especially since it's not part of infographics, or need any utils from panel component. It'd clean this code up a bit. Let me know if you have different opinion and have any questions !